### ISSUE TITLE
Create an Easy to Install Demo App

### ISSUE THREAD

**Issue by jbeorse** [opendatakit/tool-suite-X#123]
An easy-to-install demo app that will let new users experience the tools and demonstrate to others the features and benefits of ODK-X

From the convening Roadmap: https://docs.google.com/document/d/1_4x9x3eHxo4qevBRlpmCC9L2bVlUkAQuAlidT8wJDzI/edit

**Comment by maprehensive** [comment]
The server has been setup at: https://demo.odk-x.com/ 

Forms delivered from the server need to edited to match those in the Tables sample app.
https://docs.opendatakit.org/odk-x/tables-sample-app/

**Comment by maprehensive** [comment]
Initial sync can take up to 15 minutes. 
Review content in forms, and scope stripping full form download in the demo app.

**Comment by maprehensive** [comment]
Have a possible issue with the demo instance at https://tables-demo.odk-x.org

When attempting to sync the following error is returned:

![Screenshot_20201114-153620](https://user-images.githubusercontent.com/1215441/99143450-28684400-2690-11eb-98fb-fb5bac447bf1.png)

Am using Tables / Services version 2.1.6 rev 238.

@linl33 can you advise if this is a client or server issue, and possible solutions?

**Comment by e2themillions** [comment]
This could be caused by empty file? Ref "Improve error message for empty files" issue #58

**Comment by maprehensive** [comment]
@wbrunette do you think this issue should include getting a sample Tables app into the Play Store?

That was raised at a PMC meeting previously, and I can't recall if it was confirmed.

**Comment by wbrunette** [comment]
@maprehensive yes that is still the goal. However, there are details that we are working through (other things have taken priority). We are working towards this, but we still have some open things such as namespace and anonymous permissions (which was covered in today's meeting).

**Comment by maprehensive** [comment]
@wbrunette thanks – will leave this issue open then and table discussion of scope for a future meeting.
